### PR TITLE
Ensure backend supports combining labels and search terms

### PR DESCRIPTION
**Pull request by hallvors**
Partial fix for #795

### CODE REVIEW

File: webcompat/api/endpoints.py [hunk 1]
```diff
@@ -120,7 +120,10 @@ def get_issue_category(issue_category):
     if issue_category in category_list:
         # add "status-" before the filter param to match the naming scheme
         # of the repo labels.
-        params['labels'] = 'status-' + issue_category
+        params.add('labels', 'status-'+issue_category)
```
Comment (miketaylr) [inline webcompat/api/endpoints.py]: nit: whitespace around `+` operator

File: webcompat/api/endpoints.py [hunk 2]
```diff
@@ -120,7 +120,10 @@ def get_issue_category(issue_category):
...
+        # Turns out the GitHub API considers &labels=x&labels=y an OR query
+        # &labels=x,y is an AND query. So we need to join the labels with a comma
```
Comment (miketaylr) [inline webcompat/api/endpoints.py]: nit: line length

**Comment by miketaylr** [comment]
r+ with the style nits addressed.

**Comment by hallvors** [comment]
Ready to merge, I think

**Comment by hallvors** [comment]
(sorry, just did those small edits directly on the github website and forgot to mention the issue number in the commit message - I will pull, squash and reword.. Some time savers aren't.. :()

**Comment by miketaylr** [comment]
:balloon:

**Pull request merged**